runelite-mixins/src/main/java/net/runelite/mixins/RSClientMixin.java

```diff
@@ -834,6 +846,25 @@ public RSItemContainer getItemContainer(InventoryID inventory)
 		return (RSItemContainer) itemContainers.get(inventory.getId());
 	}
 
+	@FieldHook("changedItemContainers")
```

Comment:
Can we move this back to itemContainerMixin? I know the fields are in client but the client mixin is crammed-jam-packed as it is, and it really really belongs there in my opinion (or in a script/widget mixin? but that's less intuitive)